assets/js/admin/components.js

```diff
@@ -0,0 +1,211 @@
+// This file contains only import, configuration and initialization of components
+//import addDocumentEventListeners from '../utilities/components/add-document-event-listeners';
```

Comment:
Can we rather keep this empty and add stuff as we need it?

Reply:
components updated here https://github.com/netgen/media-site/commit/c4c897f58797d6772823557b30c8fcde98037e10